packages/app-desktop/app.ts

```diff
@@ -492,7 +493,9 @@ class Application extends BaseApplication {
 		if (Setting.value('startMinimized') && Setting.value('showTrayIcon')) {
 			// Keep it hidden
 		} else {
-			bridge().window().show();
+			// bridge().window().show();
```

Comment:
What would happen when the other condition is true? `if (Setting.value('startMinimized') && Setting.value('showTrayIcon'))`

Reply:
I have fixed this bug/issue of splash not hidden for `startMinimized/showTrayIcon`. With this new change, splash screen will be shown when app is starting in tray minimized. Not sure if this is expected/desired behaviour.

Reply:
If I start the app minimized (in tray), I am pretty sure that I don't want to see a splash screen.

Reply:
I agree. @palerdot please don't show it when starting minimised

Reply:
Settings are not available in `main process`/`ElectronAppWrapper`. It is available in renderer process only at this point - https://github.com/laurent22/joplin/blob/dev/packages/app-desktop/app.ts#L352. If the app starts with splash screen hidden by default, and enabling it for normal startup when settings are available, the app still starts without any splash screen for around 99% of time (several seconds), and just shows for less than a second, more as a flicker, before it is hidden again and main app window is shown.

This might be related to https://github.com/laurent22/joplin/issues/7767#issuecomment-1515887247.